src/Typography/Font.php

```diff
@@ -11,6 +11,8 @@ class Font implements FontInterface
     protected float $size = 12;
     protected float $angle = 0;
     protected mixed $color = '000000';
+    protected mixed $strokeWidth = 1;
```

Comment:
By default the text should not be outline. So I guess we need to change this to `0`?